plugins/importexport/native/filter/NativeXmlIssueFilter.inc.php

```diff
@@ -74,8 +73,21 @@ function handleElement($node) {
 		$issue->setCurrent($node->getAttribute('current'));
 		$issue->setAccessStatus($node->getAttribute('access_status'));
 
-		$issueDao->insertObject($issue);
-		$deployment->setIssue($issue);
+		$foundIssue = null;
+		foreach ($node->getElementsByTagName('issue_identification') as $n) {
+			$searchIssue = $issueDao->newDataObject();
+			$this->parseIssueIdentification($n, $searchIssue, false);
```

Comment:
I tried doing this a couple of times, but each time ended up looking a lot like the current code.  I'll push a halfway-to-this change; let me know what you think.